### DISCUSSION TITLE
Modal spans to the edges of the screen

### DISCUSSION THREAD

**Discussion by filip-dahlberg** [chakra-ui/chakra-ui#2809] [General] [answered]
## Problem Description
I have problem with the Modal. I'm successfully popping it, but it spans to the edges of the screen. Also the background and its container are not appearing, see attached screenshot below. The code is aslo attached below. Would appreciate any help.

## Screenshot of Modal spanning to the edges of the screen
<img width="1280" alt="Screenshot 2020-12-17 at 07 32 08" src="https://user-images.githubusercontent.com/37364201/102452656-fd7c6180-403a-11eb-8b93-be2274e2e1be.png">

### Modal Code 
`import React from 'react'
import {
    Modal,
    ModalOverlay,
    ModalContent,
    ModalHeader,
    ModalFooter,
    ModalBody,
    ModalCloseButton,
    useDisclosure,
    FormControl,
    FormLabel,
    Input,
  } from "@chakra-ui/react"

import {Button} from "@chakra-ui/core"
const ModalSubscribe = props => {
    const { isOpen, onOpen, onClose } = useDisclosure()

    const initialRef = React.useRef()
  
    return (
        <>
          <Button onClick={onOpen}>Open Modal</Button>
          <Modal
            initialFocusRef={initialRef}
            isOpen={isOpen}
            onClose={onClose}
            size="md"
          >
            <ModalOverlay />
            <ModalContent>
              <ModalHeader>Let me know when in stock</ModalHeader>
              <ModalCloseButton />
              <ModalBody pb={6}>
                <FormControl>
                  <FormLabel>Email</FormLabel>
                  <Input ref={initialRef} placeholder="Email" />
                </FormControl>
          
                <FormControl mt={4}>
                  <FormLabel>Phone number</FormLabel>
                  <Input placeholder="Phone number" />
                </FormControl>
              </ModalBody>
          
              <ModalFooter>
                <Button colorScheme="blue" mr={3}>
                  Save
                </Button>
                <Button onClick={onClose}>Cancel</Button>
              </ModalFooter>
            </ModalContent>
          </Modal>
        </>
    )
}

export default ModalSubscribe;`

**Comment by filip-dahlberg** [comment]
It seems to be due to mixing chakra-ui/core with chakra-ui/react

**Comment by filip-dahlberg** [comment, marked as answer]
My problem was solved by migrating from chakra-ui/core to chakra-ui/react. Reading further about it, chakra-ui/core seems to be used un earlier versions of chakra-ui while chakra-ui/react is the new version. Hence it makes sense that mixing these versions can result in trouble.

**Reply by with-heart** [reply]
Glad you were able to sort that out!
